### PR TITLE
reset fail user fail counter on password update

### LINKED ISSUE #6197: login_fail_counter not reset when an admin resets a password
When you reset a user's password in Admin-> Users
https://github.com/openemr/openemr/blob/995e8779312a94faad2a1486343604fca86040c7/interface/usergroup/user_admin.php#L307

There is nothing that will reset the `login_fail_counter` in `users_secure`

Right now, you have to manually go into the table and reset the field to 0 for a user to log in again if you have max reties enabled and admin reset a password.

### PR DESCRIPTION
**Pull request by sorvani**
This fixes #6197

### CODE REVIEW

**Review by stephenwaite — APPROVED** [review]

**Comment by stephenwaite** [comment]
nice

File: src/Common/Auth/AuthUtils.php [hunk 1]
```diff
@@ -642,6 +642,7 @@ public function updatePassword($activeUser, $targetUser, &$currentPwd, &$newPwd,
             $updateParams = array();
             $updateSQL = "UPDATE `users_secure`";
             $updateSQL .= " SET `last_update_password` = NOW()";
+            $updateSQL .= ", `login_fail_counter` = 0";
```
Comment (sjpadgett) [inline src/Common/Auth/AuthUtils.php:645]: Should be set to our default as if a new user right?

Comment (stephenwaite) [reply src/Common/Auth/AuthUtils.php:645]: this is in the update user block

Comment (sorvani) [reply src/Common/Auth/AuthUtils.php:645]: @stephenwaite is correct, this is part of the update user section.

**Review by sjpadgett — APPROVED** [review]
Okay! It's really great to see you folks putting up fixes so you have my thanks....

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
nice!

2DV-2(6)

btw, if interested in another related improvement in this feature (set a timeout to reset the counter), which have discussed here: https://github.com/openemr/openemr/issues/5627

**Pull request merged**